src/pages/edit_resource/ResourceFormActions.tsx

```diff
@@ -1,40 +1,53 @@
 import React, { FC, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import styled from 'styled-components';
-import { Button, CircularProgress, Typography } from '@material-ui/core';
+import { Button } from '@material-ui/core';
 import { publishResource } from '../../api/resourceApi';
 import { getStepLabel, ResourceFormStep, ResourceWrapper } from '../../types/resource.types';
-import { StyledSchemaPart } from '../../components/styled/Wrappers';
+import { StyledContentWrapperMedium, StyledSchemaPart } from '../../components/styled/Wrappers';
 import ErrorBanner from '../../components/ErrorBanner';
 import { useHistory } from 'react-router-dom';
 import { useFormikContext } from 'formik';
 import ArrowForwardIcon from '@material-ui/icons/ArrowForward';
 import ArrowBackIcon from '@material-ui/icons/ArrowBack';
 
+const PageWidthThresholdForButtons = '45rem';
```

Comment:
Denne verdien tok jeg med prøving og feiling. small ble for lite, og jeg fant ikke noen enkel og grei løsning på å gjøre wrappingen automatisk når skjermen blir for smal